data/config-schema/src/main/resources/net/corda/schema/configuration/membership/1.0/corda.membership.json

```
@@ -37,6 +37,13 @@
           "description": "The maximum duration in minutes to wait for a verify member request.",
           "$ref": "#/$defs/ttl",
           "default": 20
+        },
+        "waitForMgmSession": {
+          "description": "The maximum duration in minutes to wait for a member to establish connection with the MGM.",
```

```suggestion
          "description": "The maximum duration in minutes to wait for a member to establish connection with the MGM. This only has effect on time waiting for an MGM to establish a session with a member as part of processing a registration request.",
```
we should clarify it doesn't have an effect on all communication with the MGM.